tests/main/interfaces-personal-files/task.yaml

```diff
@@ -8,36 +8,30 @@ prepare: |
     . "$TESTSLIB/snaps.sh"
     install_local test-snapd-sh
 
-    # shellcheck source=tests/lib/files.sh
-    . "$TESTSLIB/files.sh"
-
     # Fist layer of dirs and files
-    ensure_file_exists_backup_real /root/.testfile1
-    ensure_file_exists_backup_real /root/testfile1
-    ensure_dir_exists_backup_real /root/.testdir1
-    ensure_dir_exists_backup_real /root/testdir1
+    "$TESTSTOOLS"/fs-state mock-file /root/.testfile1
+    "$TESTSTOOLS"/fs-state mock-file /root/testfile1
+    "$TESTSTOOLS"/fs-state mock-dir /root/.testdir1
+    "$TESTSTOOLS"/fs-state mock-dir /root/testdir1
 
     # Second layer of dirs and files
-    ensure_file_exists_backup_real /root/.testdir1/.testfilé2
-    ensure_file_exists_backup_real "/root/.testdir1/test file2"
-    ensure_dir_exists_backup_real /root/.testdir1/.testdir2
+    "$TESTSTOOLS"/fs-state mock-file /root/.testdir1/.testfilé2
```

Comment:
Unrelated to the change but the accent over `e` is probably an undesired accident.